common/src/test/java/org/opensearch/ml/common/transport/model/MLUpdateModelInputTest.java

```diff
@@ -85,7 +84,7 @@ public class MLUpdateModelInputTest {
             "{\"model_type\":\"testModelType\",\"embedding_dimension\":100,\"framework_type\":\"SENTENCE_TRANSFORMERS\",\"all_config\":\""
             +
             "{\\\"field1\\\":\\\"value1\\\",\\\"field2\\\":\\\"value2\\\"}\"},\"connector_id\":" +
-            "\"test-connector_id\",\"connector\":{\"description\":\"updated description\",\"version\":\"1\",\"parameters\":{},\"credential\":{}}}";
+            "\"test-connector_id\",\"interface\":{}}";
```

Comment:
Add some test case for a non-empty interface ?

Reply:
See UTs in task runner and trasport prediction